src/components/ec-input-field/ec-input-field.vue

```diff
@@ -109,13 +109,13 @@ $ec-input-field-border: $level-5-placeholders !default;
   }
 
   &__label {
-    @include caption-text-bold;
+    @include input-text-bold;
```

Comment:
We do have typography named "Input Label". We should use that name and not using "bold" in the name.